README.md

```diff
@@ -1,76 +1,5 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# FCC Test Suite Template
```

Comment:
- [ ] It seems to be your project is based on a template. While it's good to give recognition to the original author, keep in mind that you still need to document your code professionalism, so other people can understand what your project is about. You can always give recognition through an Acknowledgment section.